src/Forms/ElementalAreaField.php

```diff
@@ -208,4 +212,55 @@ public function performReadonlyTransformation()
             ->setName($this->getName())
             ->addExtraClass('elemental-area--read-only');
     }
+
+    /**
+     * {@inheritdoc}
+     */
```

Comment:
This is the same as not having a doc block at all, you may as well leave it out and save the lines

Reply:
> It'd be good to return `$this` for chaining, and also these methods could use doc blocks

😆 I'll remove them again 🙂

Reply:
Love me a transparent doc block